### PR TITLE
Add async snapshot example

### PR DESCRIPTION
**Pull request by diffuse**
Also update ArloCamera request_snapshot docstring

### CODE REVIEW

File: pyaarlo/camera.py [hunk 1]
```diff
@@ -635,6 +635,12 @@ def _take_idle_snapshot(self):
                           headers={"xcloudId": self.xcloud_id})
 
     def request_snapshot(self):
+        """Requests a snapshot from the camera without blocking.  
+
+        The snapshot can be handled with callbacks registered to
+        LAST_IMAGE_SRC_KEY - lastImageSource starting with snapshot/, and/or
```
Comment (twrecked) [inline pyaarlo/camera.py:641]: I think this line should be
   `lastImageSource starting with snapshot/, or capture/`

**Comment by twrecked** [comment]
Many thanks for this. I just added a comment, I think the doc should be:

   `lastImageSource starting with snapshot/, or capture/`

**Comment by diffuse** [comment]
> Many thanks for this. I just added a comment, I think the doc should be:
> 
> `lastImageSource starting with snapshot/, or capture/`

My pleasure.  To confirm, this is what you'd like the updated docstring to look like?:
```python
"""Requests a snapshot from the camera without blocking.
The snapshot can be handled with callbacks registered to 
LAST_IMAGE_SRC_KEY - lastImageSource starting with snapshot/, or capture/
LAST_IMAGE_DATA_KEY - presignedLastImageData containing the image data.
"""
```

Or without the `LAST_IMAGE_SRC_KEY` completely?

I noticed I left an unecessary f-string on async-snapshot:55 too, so I'll remove the 'f' as well.

**Comment by twrecked** [comment]
Sorry I wasn't clear, please leave in the `LAST_IMAGE_SRC_KEY`.

That new comment looks good.

**Comment by diffuse** [comment]
No problem, updated!

**Pull request merged**